### PR TITLE
Set wide shell width in case of long db names, cluster also

### PR DESCRIPTION
**Pull request by dbouchillon**
Fixes ZPS-944

Set wide shell width so that lines do not wrap around

### CODE REVIEW

**Comment by dbouchillon** [comment]
bump

**Comment by dbouchillon** [comment]
bump

File: ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py [hunk 1]
```diff
@@ -517,7 +522,10 @@ def build_command_line(self, sqlConnection):
         jobs_sqlConnection.append("'|CurrentRunStatus:'$job.CurrentRunStatus;")
         jobs_sqlConnection.append("}}")
         script = "\"& {{{}}}\"".format(
-            ''.join(getSQLAssembly(sqlConnection.version) + sqlConnection.sqlConnection + jobs_sqlConnection))
+            ''.join([BUFFER_SIZE] +
+                    getSQLAssembly(sqlConnection.version) +
+                    sqlConnection.sqlConnection +
+                    jobs_sqlConnection))
```
Comment (yichi-lu) [inline ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py]: got this while modeling: ERROR zen.PythonClient: Error in zenoss.winrm.WinMSSQL: cannot concatenate 'str' and 'list' objects.

**Comment by dbouchillon** [comment]
Weird i didn't get that when i was initially testing.  getting it now though.  updated code

**Pull request merged**